package.json

```diff
@@ -12,13 +12,16 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://bitbucket.com/uazgraduatecollege/developer-essentials"
+    "url": "git+https://github.com/uazgraduatecollege/developer-essentials"
   },
   "author": "",
   "license": "MIT",
   "bugs": {
-    "url": "https://bitbucket.com/uazgraduatecollege/developer-essentials/issues"
+    "url": "https://gighub.com/uazgraduatecollege/developer-essentials/issues"
   },
-  "homepage": "https://bitbucket.com/uazgraduatecollege/developer-essentials/#readme",
-  "dependencies": {}
+  "homepage": "https://gighub.com/uazgraduatecollege/developer-essentials/#readme",
```

Comment:
@pbarabe  Should this say github.com?

Reply:
Good catch. Yes it should. Wanna open an issue for these 2 typos?

Reply:
I noticed that the license in package.json is incorrect, too.

`"license": "MIT",` should be `"license": "See license in file LICENSE,"`